test/gtests/prebuild.sh

```diff
@@ -0,0 +1,5 @@
+cd ../deps/googletest
+mkdir build
```

Comment:
please rename this file to `build.sh`, and:
```suggestion
mkdir -p build
```

Reply:
done